crates/full-node/sov-db/src/ledger_db/mod.rs

```diff
@@ -341,7 +341,7 @@ impl LedgerDb {
         blob: StoredDiscardedBlob,
         schema_batch: &mut SchemaBatch,
     ) -> anyhow::Result<()> {
-        schema_batch.put::<DiscardedBlobByHash>(&blob.hash, &blob)
+        schema_batch.put::<DiscardedBlobByHash>(&blob.discarded_blob.hash.0, &blob)
```

Comment:
It looks like we still only store the hash alone in the LedgerDB (unless I missed something). Is this intended?

Reply:
Never mind - just realised it's now part of the StoredDiscardedBlob